components/script/dom/serviceworkerglobalscope.rs

```diff
@@ -4,75 +4,43 @@
 
```

Comment:
As suggested about serviceworkerglobalscopes, not having the parent sender, the worker instance, and the parent runtime, i have removed the code, which had interactions with them. But will need advice on the `run_with_memory_reporting` thing when listening for events, which needs a parent_sender. I think, at the very minimum, we will need a service worker object instance, to facilitate dispatching of events to them. Thoughts ?